docs/Deployment/BackupRestore.md

```diff
@@ -0,0 +1,324 @@
+Always make sure your backup strategy is tested and fits your business needs. No one solution fits all.
+For the latest updates, always feel free to ask on the BTCPay Server community channels. 
+:::
+
+
+### Restore output
+
+`btcpay-restore.sh` Output example : 
+
+```
+ℹ️  Cleaning restore directory /var/lib/docker/volumes/backup_datadir/_data/restore …
+
+🔐 Decrypting backup file …
+gpg: AES256.CFB encrypted data
+gpg: encrypted with 1 passphrase
+✅ Decryption done.
+
+ℹ️  Extracting files in /var/lib/docker/volumes/backup_datadir/_data/restore…
+postgres.sql.gz
+volumes/generated_bitcoin_wallet_datadir/
+volumes/generated_bitcoin_wallet_datadir/_data/
+volumes/generated_bitcoin_wallet_datadir/_data/mainnet/
+volumes/generated_bitcoin_wallet_datadir/_data/mainnet/.walletlock
+volumes/generated_bitcoin_wallet_datadir/_data/mainnet/wallet.dat
+volumes/generated_bitcoin_wallet_datadir/_data/mainnet/db.log
+volumes/generated_btcpay_datadir/
+volumes/generated_btcpay_datadir/_data/
+volumes/generated_btcpay_datadir/_data/FastSynced
+volumes/generated_btcpay_datadir/_data/LocalStorage/
+volumes/generated_btcpay_datadir/_data/LocalStorage/tmp/
+volumes/generated_btcpay_datadir/_data/btcpay20220512.log
+volumes/generated_btcpay_datadir/_data/tmp/
+volumes/generated_btcpay_datadir/_data/host_id_rsa
+volumes/generated_btcpay_datadir/_data/Main/
+volumes/generated_btcpay_datadir/_data/Main/settings.config
+volumes/generated_btcpay_datadir/_data/host_id_rsa.pub
+volumes/generated_btcpay_datadir/_data/host_authorized_keys
+volumes/generated_btcpay_datadir/_data/key-47ec2b82-ed01-45ef-8a63-390470cc10e5.xml
+[...]
+volumes/generated_tor_torrcdir/
+volumes/generated_tor_torrcdir/_data/
+volumes/generated_tor_torrcdir/_data/torrc
+volumes/generated_tor_torrcdir/_data/torrc-2
+
+ℹ️  Restoring database …--
+-- PostgreSQL database cluster dump
+--
+SET default_transaction_read_only = off;
+SET
+SET client_encoding = 'UTF8';
+SET
+SET standard_conforming_strings = on;
+SET
+--
+-- Drop databases (except Postgres and template1)
+--
+DROP DATABASE btcpayservermainnet;
+DROP DATABASE nbxplorermainnet;
+ERROR:  database "btcpayservermainnet" is being accessed by other users
+DETAIL:  There is 1 other session using the database.
+
+[...]
+
+--
+-- PostgreSQL database dump complete
+--
+--
+-- PostgreSQL database cluster dump complete
+--
+
+ℹ️  Cleaning up …
+
+✅ Restore done
+```
+
+## What isn't included and should be considered before taking this backup approach. 
+
+### Lightning channel backup. 
+
+:::warning
+It has a high chance of failure in a disaster recovery scenario, where you may do a backup once per night and need to restore that one backup.
+For instance, the Lightning static channel backup should be watched by a script and copied over to a remote server to ensure you always have the latest state available. The scripts will be sufficient in a migration case, where the shutdown of the old and the start of the new server happen directly. 
```

Comment:
I would very much appreciated a Link to the SCB solution of LND at this point.

Reply:
We will add this as a separate backup script. This is to be done.

Reply:
If you do happen to more things to consider when running the backup LMK. 
I think it's wise, just as I stated when you start reading this document, that BTCPay in it self can't and won't be responsible for it in anyway. The more we can make the users aware of the better and less questions will be raised.